modules/tools/treesitter/config.el

```diff
@@ -2,24 +2,12 @@
 
 (use-package! tree-sitter
   :when (bound-and-true-p module-file-suffix)
-  :hook (prog-mode . tree-sitter-mode)
+  :hook (doom-first-buffer . global-tree-sitter-mode)
```

Comment:
I would keep this as `prog-mode` as currently emacs tree sitter does not support any non prog mode languages and we don't want to load all of tree sitter for an org capture entry

Reply:
I see, switched to `prog-mode(-hook)` & `turn-on-tree-sitter-mode` instead.